pkg/server/provider.go

```diff
@@ -27,4 +27,8 @@ var (
 		ProvideTracerProvider,
 		NewServer,
 	)
+	GraphQLServerProviderSet = wire.NewSet(
```

Comment:
if this is `GraphQLServerProviderSet`, shouldn't `ServerProviderSet` be something like `HTTPServerProviderSet` or `RestServerProviderSet`?

Reply:
Yap, we should rename the REST one to `RESTServerProviderSet`